remotespark/livyclientlib/livysession.py

```diff
@@ -90,11 +89,14 @@ def create_sql_context(self):
         self.ipython_display.writeln("Creating HiveContext as 'sqlContext'")
         command = self._get_sql_context_creation_command()
         try:
-            command.execute(self)
+            (success, out) = command.execute(self)
         except LivyClientTimeoutError:
             raise LivyClientTimeoutError("Failed to create the SqlContext in time. Timed out after {} seconds."
                                          .format(self._wait_for_idle_timeout_seconds))
-        self.created_sql_context = True
+        if success:
+            self.created_sql_context = True
+        else:
+            raise ValueError("Failed to create the SqlContext.\nError {}".format(out))
```

Comment:
I would put a semicolon after Error and quote the actual error:
`raise ValueError("Failed to create the SqlContext.\nError: '{}'".format(out))`

What do you think?